apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2023-05-24T17:19:40Z"
-  build_hash: 6542e04cc357215c91df94e747e3d412fb26df97
-  go_version: go1.19.4
-  version: v0.26.1-3-g6542e04
-api_directory_checksum: 399f9b6fb5675139a83fc87a26271448ad5d9584
+  build_date: "2023-10-05T20:54:20Z"
+  build_hash: 7445de38211639a12e79992d154adab6e60f01fd
+  go_version: go1.20.3
+  version: v0.27.1-1-g7445de3
```

Comment:
This works with runtime 0.27 only

Reply:
Could you guide on how to move back to 0.27? Is it decided in code-generator repo - go.mod?

Reply:
I mean - using the code-generator 0.27.1 is correct here. You just need to pin to the latest ACK runtime in `go.mod` + do a `go mod tidy`

Reply:
Thanks @a-hilaly . I made the change. Most of the tests are passing. `/test cloudwatchlogs-metadata-file-test` failure seems to be unrelated though